### PR TITLE
Update styles for multiple select elements. Add example of small and fieldset element.

### LINKED ISSUE #104: Multiple select has arrow in the middle
Add `multiple` to the select tag in the demo page to see. Perhaps it would be helpful to have a multiple select element in the demo page also.

### PR DESCRIPTION
**Pull request by olgark**
Ran yarn build before pushing. Actual changes were made in _forms.css and index.html.

In addition to the style updates, I wanted to format the explanation text for the multiple select a little more nicely, and that was a good opportunity to introduce the `small` element to the page (no styling was needed for this in my opinion). I hope that was ok!

I also added an example and styles for a fieldset element. I moved some of the existing elements into the fieldset to make sort of a login form example.

Fixes #104

### CODE REVIEW

File: src/parts/_forms.css [hunk 1]
```diff
@@ -128,4 +134,14 @@ textarea:disabled {
 
 ::placeholder {
   color: var(--form-placeholder);
+}
+
+fieldset {
```
Comment (kylejrp) [inline src/parts/_forms.css]: This should have a margin to match the other form elements. It looks a bit off on the demo page:
image

You'll probably want the same margin values as the `input, button, textarea, select` selector.

**Comment by kylejrp** [comment]
Oh, and this probably could have been separated into three different pull requests:
1. Support for `select multiple`
2. Added `small` to demo page
3. Support for `fieldset/legend`

This just makes it easier to review the pull request and allows for a cleaner commit history. But don't worry about it this time! 😄

File: src/parts/_forms.css (hunk 1 above)
Comment (olgark) [reply src/parts/_forms.css]: Good call, I noticed it had a left margin imposed by the user agent stylesheet, so I zeroed out the margin and added a bottom margin to match the other elements. I also updated the padding to be consistent with the other elements. Thanks! 

Question: When someone comments on your PR and you push changes to resolve the issue, is the code author supposed to hit the Resolve button, or is the commenter supposed to? Thanks for being so helpful!

Comment (kylejrp) [reply src/parts/_forms.css]: Perfect, that's exactly what I was looking for. Great work!

I think it's fair for you to press the "Resolve conversation" button when you've fixed the issue, especially if it doesn't require more review. I think intended that this was resolved once you made the requested change. If anything new came up, I'd start a new review.

Other projects might have this called out in their contributing guidelines, but we don't have one yet. 😅

**Pull request merged**